script/RegisterL1Bridge.s.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Script, stdJson} from "forge-std/Script.sol";
+import {IERC20Receive} from "src/interfaces/IERC20Mint.sol";
+import {Constants} from "test/Constants.sol";
+
+contract RegisterL1Bridge is Script, Constants {
```

Comment:
Do you think for this and other scripts, we can add a few sentence comment at the top of each documenting a bit what's going on? Something along the lines of "This script should be run on the L2 by the owner of the token in order to register the blah blah blah as required by wormhole for blah blah blah" or whatever? I'm having a little trouble orienting myself as-to the purpose of each script, and those comments could go a long way toward making it more clear.

Reply:
Yes, this would definitely be helpful

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/8/commits/fc68660afa4e4a251a9a47c3cf880ee88c44d58f